generated/ridp/post_answer_questions.json

```diff
@@ -0,0 +1,34 @@
+{
+  "session_id": "AB783917E63E4CA345448C600928D632.pidd1v-1304180857460210166972210",
```

Comment:
is this actually part of the POST, or is it in a cookie? I thought it's usually in a cookie?